reproduction/bimanual/bimanual_iiwa_example.ipynb

```diff
@@ -104,13 +104,40 @@
     "iris_options.iteration_limit = 5\n",
     "iris_options.termination_threshold = -1\n",
     "iris_options.relative_termination_threshold = 0.02\n",
+    "iris_options.num_collision_infeasible_samples = 1\n",
     "CORE_CNT = int(mp.cpu_count()/2) # you may edit this\n",
     "\n",
     "filterCollsionGeometry(scene_graph, sg_context)\n",
     "\n",
     "seed_points = getConfigurationSeeds()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Load pre-generated regions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with open(os.path.join(GcsDir(), 'data/bimanual/iris_regions.reg'), 'rb') as f:\n",
```

Comment:
should this still be first if that file is not in the repo?

Reply:
Fair point.